### PR TITLE
feat: configure statuboard

### PR DESCRIPTION
**Pull request by bjohansebas**
This revives the statusboard, bringing in the changes made in #1, using modern GitHub Actions

preview: https://bjohansebas.github.io/statusboard/

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      url: ${{ steps.deployment.outputs.page_url }}
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+    - name: Setup Pages
+      uses: actions/configure-pages@v5
+    - uses: actions/setup-node@v4
+      with:
+        node-version: 12
```
Comment (bjohansebas) [inline .github/workflows/build.yml:33]: It was the latest version of Node.js that could run the statusboard.

**Review by IamLizu — APPROVED** [review]
Looks okay to me :+1

File: .github/workflows/build.yml (hunk 1 above)
Comment (wesleytodd) [reply .github/workflows/build.yml:33]: Yeah this was written a long time ago. We should spend some cycles getting that fixed in [the upstream](https://github.com/pkgjs/statusboard).

**Review by wesleytodd — APPROVED** [review]

**Comment by wesleytodd** [comment]
https://github.com/expressjs/statusboard/actions/runs/11682395658

Looks like it is failing despite the tests passing. 404s, so my guess is something is broken in the upstream.

**Comment by wesleytodd** [comment]
Yep, looks like the same errors are happening here: https://github.com/pkgjs/statusboard/pull/20/checks

**Comment by wesleytodd** [comment]
Looks also like with the new gh-pages action we need to use the `gh-pages` branch to build this. It has been too long since I looked at this code, so I don't remember enough details to give a quick suggestion on a fix, but also I didn't use the gh-pages actions IIRC. Since this was already broken, I dont think we need to fix it on a schedule, but I can try to pay attention so if you have a chance to look at what is broken.

**Comment by bjohansebas** [comment]
I think the error is happening because deployments are only allowed on the gh-pages branch, so we need to add the master branch for deployment. The GitHub Pages environment should be configured like this:

image

https://github.com/express/statusboard/settings/environments/

**Pull request merged**